reference/tax_settings.v3.yml

```diff
@@ -146,7 +146,19 @@ components:
         should_subtract_store_tax:
           default: true
           type: boolean
-          description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.    
+          description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.
+        should_use_geolocation_to_determine_guest_shopper_tax_zone:
```

Comment:
I know this has probably been discussed in the PR when introducing this field name, but this name is a mouthful.

Would something shorter like `determine_tax_zone_by_geolocation` be easier to digest here?

Reply:
I agree the variable name is quite long, but also think that it is important to keep the context as well. I personally think that keeping `guest_shopper` or related information somewhere in the variable makes it more straight forward for users.
Happy to change it to something else if we get to decide on another variable that is more concise and is clear

Reply:
Yes, we did consider how we could shorten this naming originally but we did not identify a better option.

Reply:
Exploring the language we're using in these descriptions, would this make sense: `should_use_guest_shopper_geolocation`? I'm not completely sold on it, but it does tighten things up.

The flaw is that we're not describing what this is being used for 😆 There could be a fair argument that this is a detail that can be described instead.

Reply:
Does it need to be tight?. I feel like `should_use_geolocation_to_determine_guest_shopper_tax_zone` is very descriptive and is non-ambiguous.

Reply:
It doesn't need to be.

I was also exploring trying to "proper noun" this feature a bit more, but I'm coming up blank.